src/PowerFlowData.jl

```diff
@@ -84,6 +84,7 @@ struct PowerFlowData{
 } <: PowerFlowContainer
     bus_lookup::Dict{Int, Int}
     branch_lookup::Dict{String, Int}
+    transformer_3w_lookup::Dict{String, Vector{String}}
```

Comment:
What is this? Explain in the docstring

Reply:
We need a way to map the branch results back to the 3W transformer component. The 3W transformer is an AC branch but is represented by 3 branches in the calculation at the data level and Ybus level. So if we need to know the results for the 3W transformer, we need to backtrace them from the 3 branches.